### PR TITLE
libtirpc: don't build host shared libraries

### PR DESCRIPTION
**Pull request by neheb**
Maintainer: @Andy2244

### CODE REVIEW

**Comment by neheb** [comment]
```
configure: error: Missing library functions needed for IPv6.
```

what?

**Comment by neheb** [comment]
testing on Fedora 35 (has host libtirpc) and Alpine Linux (no host libtirpc) and ldd shows this is working as intended. Wonder why it's failing here.

File: net/samba4/Makefile [hunk 1]
```diff
@@ -5,6 +5,7 @@ PKG_NAME:=samba
 PKG_VERSION:=4.14.12
 PKG_RELEASE:=$(AUTORELEASE)
 
+
```
Comment (feckert) [inline net/samba4/Makefile:8]: Is that intentional?

Comment (neheb) [reply net/samba4/Makefile:8]: Yeah it's just a test commit to get it to build. samba4 relies on libtirpc/host I believe.

**Pull request merged**